cmd/root.go

```diff
@@ -115,8 +131,7 @@ func setRootPath(path string) {
 	// get absolute path to .git dir (project root)
 	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
 
-	outputBytes, err := cmd.CombinedOutput()
-	check(err)
+	outputBytes, _ := cmd.CombinedOutput()
```

Comment:
we don't check the error because we either checked it in the global `PersistentPreRun` or don't care about git (i.e., in `help` or `version`)